content/en/docs/additional-concepts/resourcequotas-and-limitranges/_index.md

```diff
@@ -9,25 +9,6 @@ In this lab, we are going to look at ResourceQuotas and LimitRanges. As {{% para
 For this lab to work it is vital that you use the namespace `<username>-quota`!
 {{% /alert %}}
 
-{{% onlyWhen rancher %}}
```

Comment:
I think this part is still needed at mobi in order to create the resource limits.

Reply:
I will verify this in Mobis Rancher env and make an update to this section. It for sure looks a bit different with the new Rancher version.

Reply:
Are you going to update the section in this PR or would you like to merge it as it is?

Reply:
I'm gonna merge it and leave it like that. I'll check if anything is needed again during preparation for the next mobi training.